test/sql/misc.test.lua

```diff
@@ -79,3 +79,30 @@ box.execute('SELECT a;')
 diag = box.error.last()
 box.execute('SELECT * FROM (VALUES(true));')
 diag == box.error.last()
+
+-- exclude_null + SQL correctness
+box.execute([[CREATE TABLE j (s1 INT PRIMARY KEY, s2 STRING, s3 VARBINARY) WITH ENGINE='vinyl';]])
```

Comment:
sql/ suit is ran using both engines: at the start default engine is set
`_ = box.space._session_settings:update('sql_default_engine', {{'=', 2, engine}})
`
So you don't have to specify vinyl engine.